stuffer/s2n_stuffer.h

```diff
@@ -22,6 +22,8 @@
 
 #include "utils/s2n_blob.h"
 
+#define MIN_STUFFER_GROWTH_IN_BYTES 1024
```

Comment:
I think this should probably start with `S2N`

Reply:
Done